packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,523 @@
+>**Note:** The examples in this guide use Swift 5, SwiftUI, and a target of iOS 15 or later. The SwiftUI portions pass state information using multiple bindings, consider using more robust ways of sharing and updating data in production apps. For more information, see the [Apple developer documentation for state and data flow in SwiftUI](https://developer.apple.com/documentation/swiftui/state-and-data-flow/).
+
+The code in this guide manages the sign-on flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).
```

Comment:
That's a great question to which I don't know the answer. I went back and forth with trying it as it's own note, incorporated into the previous note, the way it ended up, and even having that information in different places. The reality is that both the Note and that paragraph don't belong in this section. They belong in a place that gives information pertinent to all iOS code in the Guide... but no such place exists.

The combined note looked far too long and was hard to read, even when I put that part that's text above in a separate paragraph of the note. Two notes also looks odd and has the danger of being glossed over because it's so light.

I did it this way because the first note refers to conditions that are important and independent specific implementation details (equivalent to version of .Net and target OS.) In some ways the second part of that note on sharing information almost doesn't belong The second paragraph is about the how of the code.

It may be that only the information through "iOS 15 and later" should be note:


>**Note:** The examples in this guide use Swift 5, SwiftUI, and a target of iOS 15 or later.

The code in this guide manages the sign-on flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).

The SwiftUI portions pass state information using multiple bindings, consider using more robust ways of sharing and updating data in production apps. For more information, see the [Apple developer documentation for state and data flow in SwiftUI](https://developer.apple.com/documentation/swiftui/state-and-data-flow/).

Reply:
I've made changes that reflect the shorter note.

Reply:
nice. That's consistent now with all code specifics outside of the note